pdq/instability.go

```diff
@@ -0,0 +1,20 @@
+package pdq
+
+import (
+	"fmt"
+)
+
+func CalcInstability(outgoingDependenciesNum int, incomingDependenciesNum int) (float64, error) {
+	if outgoingDependenciesNum < 0 || incomingDependenciesNum < 0 {
+		return -1, fmt.Errorf("outgoingDependenciesNum %v, incomingDependenciesNum %v; values must be >= 0", outgoingDependenciesNum, incomingDependenciesNum)
```

Comment:
return 0 instead of -1 in case of err, since -1 can be understood as a valid metric value ..or?